### DISCUSSION TITLE
Integration question, warning in FindAllMarkers() function but it still produces a heatmap

### DISCUSSION THREAD

**Discussion by ghost** [satijalab/seurat#9117] [General]
Quick question, I've integrated two seurat objects using the FindIntegrationAnchors() and IntegrateData() functions, both seurat objects were normalized and found variable features before integration. I've been able to the downstream analysis smoothly, Scaling/PCA/UMAP , but when I use FindMarkers or FindAllMarkers to generate a DEG from the clustering, I get the warning below. I've already inspected the objects and have not found any NAs. I still get a heatmap showing up and downregulate genes per clusters but the warning persists. Could use some advice:

**markers<-FindAllMarkers(seurat_int_object, only.pos = T, min.pct = 0.25, logfc.threshold = 0.25)**
Calculating cluster 0
Warning in mean.fxn(object[features, cells.1, drop = FALSE]) :
  NaNs produced
Warning in mean.fxn(object[features, cells.2, drop = FALSE]) :
  NaNs produced
Calculating cluster 1
Warning in mean.fxn(object[features, cells.1, drop = FALSE]) :
  NaNs produced
Warning in mean.fxn(object[features, cells.2, drop = FALSE]) :
  NaNs produced
Calculating cluster 2
Warning in mean.fxn(object[features, cells.2, drop = FALSE]) :
  NaNs produced

**Comment by VincentZ3148** [comment]
exactly same. Idk but it seems that nobody cares about it( as long as it is not an error to stop code running). Lmao.
